app/screens/channel_files/empty/index.tsx

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+import React from 'react';
+import {View} from 'react-native';
+
+import FormattedText from '@components/formatted_text';
+import SearchFilesIllustration from '@components/no_results_with_term/search_files_illustration';
+import {useTheme} from '@context/theme';
+import {changeOpacity, makeStyleSheetFromTheme} from '@utils/theme';
+import {typography} from '@utils/typography';
+
+const getStyleSheet = makeStyleSheetFromTheme((theme) => ({
+    container: {
+        flex: 1,
+        alignItems: 'center',
+        justifyContent: 'center',
+        paddingHorizontal: 40,
+        maxWidth: 480,
+    },
+    title: {
+        color: theme.centerChannelColor,
+        textAlign: 'center',
+        ...typography('Heading', 400, 'SemiBold'),
+    },
+    paragraph: {
+        marginTop: 8,
+        textAlign: 'center',
+        color: changeOpacity(theme.centerChannelColor, 0.72),
+        ...typography('Body', 200),
+    },
+}));
+
+function EmptySavedPosts() {
```

Comment:
I think this was a copy/paste from another component based on the name of the component, what is different between the two components? can they be combined into one and handle the diff with props?

Reply:
Yeah I forgot to change the name, Have done the required refactoring.